aiographite/aiographitesend.py

```diff
@@ -0,0 +1,163 @@
+from .protocol import PlaintextProtocol, PickleProtocol
+from .graphite_encoder import GraphiteEncoder
+import asyncio
+import pickle
+import struct
+import socket
+import os
+import time
+from typing import Dict, Tuple, List, Callable
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AIOGraphite:
```

Comment:
Classes AioGraphiteSendException and AIOGraphite can be consistent with names. May be AIOGraphite can be named to AioGraphite?